### PR TITLE
activity assignee realtion change

### PR DESCRIPTION
**Pull request by vishaltambe70**

### CODE REVIEW

File: controllers/tag.js [hunk 1]
```diff
@@ -125,21 +125,22 @@ module.exports = {
...
             ctx.request.body.contacts.map(async (contact) => {
-              let contacttagDetails = {
-                tag: entity.id,
-                contact: contact,
-              };
-              contacttagEntry = await strapi
-                .query("contacttag", "crm-plugin")
-                .create(contacttagDetails);
-              contacttags.push(contacttagEntry);
+          let contacttagDetails = {
```
Comment (kurund) [inline controllers/tag.js]: Can you please correct the indentation.

Comment (vishaltambe70) [reply controllers/tag.js]: done!

File: controllers/tag.js [hunk 2]
```diff
@@ -125,6 +125,7 @@ module.exports = {
         if (result.error == true) {
           return ctx.badRequest(null, result.message);
         }
+
```
Comment (kurund) [inline controllers/tag.js:128]: This file should not be part of the PR
